IBAnimatable/FlipAnimator.swift

```diff
@@ -21,14 +21,13 @@ public class FlipAnimator: NSObject, AnimatedTransitioning {
   fileprivate var horizontal: Bool = false
 
   // MARK: - Life cycle
-  public init(from direction: TransitionAnimationType.Direction, transitionDuration: Duration) {
+  public init(from direction: TransitionAnimationType.Direction, duration: Duration) {
```

Comment:
Do I just need to reference this Pull Request or should I actually state the parameter name change?